src/schemas/pa-yaml/v3.0/pa.schema.yaml

```diff
@@ -435,24 +435,42 @@ definitions:
   DataSource-name:
```

Comment:
great job editing this file!
after you get done, you need to update the distributed version, which removes comments and such but is intended to be consumed by users.

Do this by running: **src\schemas\publish.cmd**